integration-test/1955-chinese-parser.py

```diff
@@ -13,6 +13,128 @@ def _tile_centre(z, x, y):
+                u"max_natsca": 300,
+                u"min_areakm": 218,
+                u"max_areakm": 1748,
+                u"min_areami": 84,
+                u"max_areami": 675,
+                u"min_perkm": 126,
+                u"max_perkm": 755,
+                u"min_permi": 78,
+                u"max_permi": 469,
+                u"min_bbxmin": -122.516667,
+                u"max_bbxmin": -122.516667,
+                u"min_bbxmax": -122.358333,
+                u"max_bbxmax": -121.733333,
+                u"min_bbymin": 37.191667,
+                u"max_bbymin": 37.575,
+                u"min_bbymax": 37.816667,
+                u"max_bbymax": 38.041667,
+                u"mean_bbxc": -122.301354,
+                u"mean_bbyc": 37.622288,
+                u"timezone": u"America\/Los_Angeles",
+                u"un_fid": 570,
+                u"pop1950": 1855,
+                u"pop1955": 2021,
+                u"pop1960": 2200,
+                u"pop1965": 2361,
+                u"pop1970": 2529,
+                u"pop1975": 2590,
+                u"pop1980": 2656,
+                u"pop1985": 2805,
+                u"pop1990": 2961,
+                u"pop1995": 3095,
+                u"pop2000": 3236,
+                u"pop2005": 3387,
+                u"pop2010": 3450,
+                u"pop2015": 3544,
+                u"pop2020": 3684,
+                u"pop2025": 3803,
+                u"pop2050": 3898,
+                u"min_zoom": 2.7,
+                u"wikidataid": u"Q62",
+                u"wof_id": 85922583,
+                u"capalt": 0,
+                u"name_en": u"San Francisco",
+                u"name_de": u"San Francisco",
+                u"name_es": u"San Francisco",
+                u"name_fr": u"San Francisco",
+                u"name_pt": u"São Francisco",
+                u"name_ru": u"Cан-франциско",
+                u"name_zh": u"旧金山",
+                u"name_ar": u"سان فرانسيسكو",
+                u"name_bn": u"সান ফ্রান্সিস্কো",
+                u"name_el": u"σαν φρανσίσκο",
+                u"name_hi": u"सैन फ्रांसिस्को",
+                u"name_hu": u"San Francisco",
+                u"name_id": u"San Francisco",
+                u"name_it": u"San Francisco",
+                u"name_ja": u"サンフランシスコ",
+                u"name_ko": u"샌프란시스코",
+                u"name_nl": u"San Francisco",
+                u"name_pl": u"San Francisco",
+                u"name_sv": u"San Francisco",
+                u"name_tr": u"San Francisco",
+                u"name_vi": u"San Francisco",
+                u"ne_id": 1159151479,
+                u"name_fa": u"سان فرانسیسکو",
+                u"name_he": u"סן פרנסיסקו",
+                u"name_uk": u"Cан-франциско",
+                u"name_ur": u"سان فرانسسکو",
+                u"name_zht": u"舊金山",
+                u"source": u"naturalearthdata.com",
+            })
+        )
+
+        # the min_zoom is 2.7 so it should appear at zoom 3 to zoom 7
```

Comment:
@nvkelso  please help confirm this.

Reply:
Testing at zoom 3 is fine... it should show up also at zoom 2... but if you're testing simply the Chinese language stuff then sometimes it's better to keep your test small... else it begins to be a "prod QA release" test rather than an simple "integration" test.

Reply:
If you want to test in the range, then I'd start at 2 (should work!).

Reply:
@nvkelso when I test zoom 2 this test failed with this error

```
FAIL: test_ne_san_francisco (integration-test.1955-chinese-parser.ChineseNameTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/pli2/Snapchat/Dev/vector-datasource/integration-test/1955-chinese-parser.py", line 135, in test_ne_san_francisco
    'source': u'naturalearthdata.com',
  File "/Users/pli2/Snapchat/Dev/vector-datasource/integration-test/__init__.py", line 1489, in assert_has_feature
    self.test_instance.assert_has_feature(z, x, y, layer, props)
  File "integration-test/__init__.py", line 1322, in assert_has_feature
    self.assertions.assert_has_feature(z, x, y, layer, props)
  File "integration-test/__init__.py", line 1151, in assert_has_feature
    "layer %r was empty)" % (properties, layer))

AssertionError: Did not find feature including properties {'kind': 'locality', 'name': 'San Francisco', 'name:zh-Hant': u'\u820a\u91d1\u5c71', 'name:zh-Hans': u'\u65e7\u91d1\u5c71', 'source': u'naturalearthdata.com', 'id': 26819236} (because layer 'places' was empty)
```

Reply:
Huh. That's configured here: 

- https://github.com/tilezen/vector-datasource/blob/master/queries/ne-places.jinja2#L41

Which should work but moving on... not important for this particular test so keep at 3.